js/.eslintrc.js

```diff
@@ -3,3 +3,6 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 module.exports = require('@attic/eslintrc');
+
+// TODO: Include this in @attic/eslintrc.
+module.exports.rules['eqeqeq'] = ['error', 'allow-null'];
```

Comment:
In spirit we are following AirBnb js style guide which does not have this exception.